bigquery/tests/unit/test__helpers.py

```diff
@@ -782,6 +782,17 @@ def _make_field(field_type, mode="NULLABLE", name="testing", fields=()):
     return SchemaField(name=name, field_type=field_type, mode=mode, fields=fields)
 
 
+def _make_connection(*responses):
```

Comment:
This file is tests for `_helpers`, not helpers for tests. I'd recommend creating a separate `tests/unit/helpers.py` if you plan to do this refactoring.

Reply:
Fixed, thanks! I was wondering why the filename looked different...